intg/src/main/java/org/apache/atlas/model/Tag.java

```diff
@@ -152,6 +152,9 @@ public AtlasClassification toAtlasClassification() throws AtlasBaseException {
         if (classification.getRemovePropagationsOnEntityDelete() == null) {
             classification.setRemovePropagationsOnEntityDelete(true);
         }
+        if (classification.getPropagate() == null) {
+            classification.setPropagate(false);
```

Comment:
IMO default for isPropagate should be `true`, I am banking on the default value set when we do not pass this flag when attaching a tag to an asset

Reply:
Feel free to try via API & verify